src/main/java/com/internet/shop/controllers/RegistrationController.java

```diff
@@ -0,0 +1,53 @@
+package com.internet.shop.controllers;
+
+import com.internet.shop.library.Injector;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.model.User;
+import com.internet.shop.service.interfaces.ShoppingCartService;
+import com.internet.shop.service.interfaces.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class RegistrationController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private static UserService userService =
```

Comment:
why service is static? 
```suggestion
    private UserService userService =
```